aws/rust-runtime/aws-hyper/src/lib.rs

```diff
@@ -71,7 +70,9 @@ pub type StandardClient = Client<DynConnector>;
 #[doc(inline)]
 pub use smithy_client::bounds::SmithyConnector;
 
-#[doc(inline)]
+/// AWS Service Client builder.
+///
+/// See [`smithy_client::Builder`] for details.
```

Comment:
These type aliases can both be `#[doc(inline)]` again once https://github.com/rust-lang/rust/issues/86120 hits stable (or subject to the MSRV policy).